src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx

```diff
@@ -159,219 +159,266 @@ Both selectors support `matchLabel` and `matchExpressions`.
-  * DoesNotExist— Pod must not include a label with the specified key. The values property must not be specified.
+  * `In`: Label's value must match one of the specified values.
+  * `NotIn`: Label's value must not match any of the specified values.
+  * `Exists`: Pod must include a label with the specified key (the value isn't important). When using this operator, you shouldn't specify the values field.
+  * `DoesNotExist`: Pod must not include a label with the specified key. The values property must not be specified.
 
-  If multiple expressions are specified, all those expressions must evaluate to true for the selector to match a pod. If both matchLabels and matchExpressions are specified, all the labels must match and all the expressions must evaluate to true for the pod to match the selector.
+If you specify multiple expressions, all those expressions must evaluate to true for the selector to match a pod. If you specify both `matchLabels` and `matchExpressions`, all the labels must match and all the expressions must evaluate to true for the pod to match the selector.
 
   </Collapser>
 </CollapserGroup>
 
 ### APM agent [#apm-agent]
 
-You've got to specify the APM agent and its version within the instrumentation CR.
-We recommend using the latest version to take advantage of the newest features available.
-
-| image  | language + version          |
-| ------ | ----------------------------|
-| dotnet | newrelic-dotnet-init:latest |
-| java   | newrelic-java-init:latest   |
-| nodejs | newrelic-node-init:latest   |
-| python | newrelic-python-init:latest |
-| ruby   | newrelic-ruby-init:latest   |
-
-Versions available for each agent:
+You've got to specify the APM agent and its version within the instrumentation CR. We recommend using the latest version to take advantage of the newest features available.
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Image
+            </th>
+            <th>
+                Language + version
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                dotnet
+            </td>
+            <td>
+                `newrelic-dotnet-init:latest`
+            </td>
+        </tr>
+        <tr>
+            <td>
+                java
+            </td>
+            <td>
+                `newrelic-java-init:latest`
+            </td>
+        </tr>
+        <tr>
+            <td>
+                nodejs
+
+            </td>
+            <td>
+                `newrelic-node-init:latest`
+            </td>
+        </tr>
+        <tr>
+            <td>
+                python
+            </td>
+            <td>
+                `newrelic-python-init:latest`
+            </td>
+        </tr>
+        <tr>
+            <td>
+                ruby
+            </td>
+            <td>
+                `newrelic-ruby-init:latest`
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+These are the versions available for each agent:
```

Comment:
Should we add these versions available info as part of the previous table, column title can be 'Versions available'?

Reply:
Done.